src/test/rpc/TransactionEntry_test.cpp

```diff
@@ -328,7 +328,7 @@ class TransactionEntry_test : public beast::unit_test::suite
             "TransactionType" : "Payment",
             "TxnSignature" : "3044022033D9EBF7F02950AF2F6B13C07AEE641C8FEBDD540A338FCB9027A965A4AED35B02206E4E227DCC226A3456C0FEF953449D21645A24EB63CA0BB7C5B62470147FD1D1",
 })",
-            "39AA166131D56622EFD96CB4B2BD58003ACD37091C90977FF6B81419DB451775",
+            "3A6E375BFDFF029A571AFBB3BC46C4F52963FAF043B406D0E59A7194C1A8F98E",
```

Comment:
why change here ?

Reply:
There are some changes to metadata hashes etc. due to the new fields.